calculator.go

```diff
@@ -1,13 +1,94 @@
 // Package calculator provides a library for simple calculations in Go.
 package calculator
 
+import (
+	"fmt"
+	"math"
+	"strings"
+)
+
 // Add takes two numbers and returns the result of adding them together.
-func Add(a, b float64) float64 {
-return a + b
+func Add(a, b float64, v ...float64) float64 {
+	r := a + b
+	for _, x := range v {
+		r += x
+	}
+	return r
 }
 
 // Subtract takes two numbers and returns the result of subtracting the second
 // from the first.
-func Subtract(a, b float64) float64 {
-	return b - a
+func Subtract(a, b float64, v ...float64) float64 {
+	r := a - b
+	for _, x := range v {
+		r -= x
+	}
+	return r
+}
+
+// Multiply takes two numbers and returns the result of multiplying them together.
+func Multiply(a, b float64, v ...float64) float64 {
+	r := a * b
+	for _, x := range v {
+		r *= x
+	}
+	return r
+}
+
+// Divide takes two numbers and returns the result of dividing the second
+// from the first.
+func Divide(a, b float64, v ...float64) (error, float64) {
```

Comment:
When we return "something and error" from a function, conventionally, the something comes first, followed by the error.

Reply:
Ah - thank you. I've seen a mix of this order in various Go code.